JS/String Compression II.js

```diff
@@ -50,27 +52,29 @@ var getLengthOfOptimalCompression = function (s, k) {
 
     for (let i = start; i < n; i++) {
       let c = s[i];
```

Comment:
It seems you are still referencing `n` in this loop which you have renamed to `strLength` if this solution still works though...magic?

Reply:
Whoops! Good catch!